rust/cardano-chain-follower/src/chain_sync_live_chains.rs

```diff
@@ -76,8 +76,14 @@ impl ProtectedLiveChainBlockList {
     fn get_block(&self, point: &Point, mut advance: i64, strict: bool) -> Option<MultiEraBlock> {
         let chain = self.0.read().ok()?;
 
-        let mut this = if strict {
+        let mut this = if strict && !point.is_fuzzy() {
+            // We can call `get` only for non fuzzy points if we want to
             chain.get(point)?
+        } else if strict && point.is_fuzzy() {
```

Comment:
```suggestion
        } else if strict  { // Strict but fuzzy point.
            // Find the required point, or one after it.
```

Must be fuzzy due to the above `if` fails.